src/sympc/api.py

```diff
@@ -135,9 +113,22 @@
     ("sympc.tensor.ShareTensor.unsqueeze", "sympc.tensor.ShareTensor"),
     ("sympc.tensor.ShareTensor.reshape", "sympc.tensor.ShareTensor"),
     ("sympc.tensor.ShareTensor.view", "sympc.tensor.ShareTensor"),
+    ("sympc.tensor.ShareTensor.expand", "sympc.tensor.ShareTensor"),
     ("sympc.tensor.static.stack_share_tensor", "sympc.tensor.ShareTensor"),
     ("sympc.tensor.static.cat_share_tensor", "sympc.tensor.ShareTensor"),
+    ("sympc.tensor.static.helper_argmax_pairwise", "sympc.tensor.ShareTensor"),
     ("sympc.tensor.ShareTensor.reshape", "sympc.tensor.ShareTensor"),
+    ("sympc.tensor.ShareTensor.repeat", "sympc.tensor.ShareTensor"),
+    ("sympc.tensor.ShareTensor.narrow", "sympc.tensor.ShareTensor"),
+    ("sympc.tensor.ShareTensor.dim", "syft.lib.python.Int"),
+    ("sympc.tensor.ShareTensor.transpose", "sympc.tensor.ShareTensor"),
+    ("sympc.tensor.ShareTensor.flatten", "sympc.tensor.ShareTensor"),
+    ("sympc.tensor.ShareTensor.expand", "sympc.tensor.ShareTensor"),
+    ("sympc.tensor.ShareTensor.roll", "sympc.tensor.ShareTensor"),
+    ("sympc.tensor.ShareTensor.shape", "syft.lib.python.Tuple"),
```

Comment:
### Part 1
Bug: `sympc.tensor.ShareTensor.expand` is added twice

### Part 2.1
Question: The other ones (`narrow`, `dim`, ... , `shape`) are to be added ... (see Part 2.2)

Reply:
I think we can remove those. Since we are not using them remotely, only locally.